test/GitcoinGovernor.t.sol

```diff
@@ -236,3 +240,190 @@ contract GitcoinGovernorProposalTest is GitcoinGovernorTest {
     assertEq(timelock.admin(), address(governor));
   }
 }
+
+contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelper {
+  address constant USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+  address constant RAD_ADDRESS = 0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3;
+
+  IERC20 usdcToken = IERC20(USDC_ADDRESS);
+  IERC20 radToken = IERC20(RAD_ADDRESS);
+
+  function setUp() public override {
+    super.setUp();
+    usdcToken = IERC20(USDC_ADDRESS);
+    radToken = IERC20(RAD_ADDRESS);
+  }
+
+  function testFuzz_OldGovernorSendsGTCAfterProposalIsDefeated(
+    uint256 _gtcAmount,
+    address _gtcReceiver
+  ) public {
+    vm.assume(_gtcReceiver != TIMELOCK && _gtcReceiver != address(0x0));
+    uint256 _timelockGtcBalance = gtcToken.balanceOf(TIMELOCK);
+    // bound by the number of tokens the timelock currently controls
+    _gtcAmount = bound(_gtcAmount, 0, _timelockGtcBalance);
+    uint256 _initialGtcBalance = gtcToken.balanceOf(_gtcReceiver);
+
+    // Defeat the proposal to upgrade the Governor
+    defeatProposal();
+
+    // Craft a new proposal to send GTC
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    string[] memory _signatures = new string [](1);
+    bytes[] memory _calldatas = new bytes[](1);
+
+    _targets[0] = GTC_TOKEN;
+    _values[0] = 0;
+    _signatures[0] = "transfer(address,uint256)";
+    _calldatas[0] = abi.encode(_gtcReceiver, _gtcAmount);
+
+    // Submit the new proposal
+    vm.prank(PROPOSER);
+    uint256 _newProposalId = governorAlpha.propose(
+      _targets, _values, _signatures, _calldatas, "Transfer some GTC from the old Governor"
+    );
+
+    // Pass and execute the new proposal
+    (,,, uint256 _startBlock, uint256 _endBlock,,,,) = governorAlpha.proposals(_newProposalId);
+    vm.roll(_startBlock + 1);
+    for (uint256 _index = 0; _index < delegates.length; _index++) {
+      vm.prank(delegates[_index]);
+      governorAlpha.castVote(_newProposalId, true);
+    }
+    vm.roll(_endBlock + 1);
+    governorAlpha.queue(_newProposalId);
+    vm.roll(block.number + 1);
+    (,, uint256 _eta,,,,,,) = governorAlpha.proposals(_newProposalId);
+    vm.warp(_eta + 1);
+    governorAlpha.execute(_newProposalId);
```

Comment:
Can this all be replaced with:

```solidity
passAndQueueProposal();
jumpPastProposalEta();
governorAlpha.execute(proposalId);
```

Reply:
No, those helpers are for the upgrade proposal. This proposal is a new one, being submitted after the upgrade proposal is defeated.

Reply:
I thought those helpers look for and use the current active proposal, regardless of the specifics of that proposal. But I now see they rely on a hardcoded `proposalId`

Reply:
Feel free to resolve this comment + below comment yourself, either by leaving as-is or generalizing those to take in a proposal ID input / removing this duplicate code

Reply:
Going to leave as-is because these are the only 3 tests that will need it. Doesn't seem worth over abstracting.